src/Core/MailTemplates/Mjml/components/mj-bw-icon-row.js

```diff
@@ -1,4 +1,12 @@
 const { BodyComponent } = require("mjml-core");
+
+const BODY_TEXT_STYLES = `
```

Comment:
Hey folks, I got some feedback from Sukhleen that the font spacing doesn’t align with the design. I think this wasn’t caught earlier because the welcome email doesn’t have multi-line text, but the org confirmation email does.

I also noticed that the style from mj-bw-icon-row-text isn’t being applied to the text. I can’t get the styling to apply through the CSS class; it might be related to how MJML renders it. 

<img width="1264" height="776" alt="Image" src="https://github.com/user-attachments/assets/7479ea95-784b-4a43-b6e1-65f25e3923f0" />

The workaround is to inject the styling directly. 

I’m open to any suggestions on how to handle this better.